tests/migration_test.go

```diff
@@ -688,6 +688,42 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 			})
 		})
+
+		Context("with a bridge network interface and MAC address specified", func() {
```

Comment:
I would also expect to see a test w/ traffic involved - e.g. traffic works fine *after* VM migration.